lib/LaTeXML/Core/Gullet.pm

```diff
@@ -212,7 +211,7 @@ our @hold_token = (
   0, 0, 0, 0,
   0, 0, 0, 0,
   0, 0, 1, 0,
-  0, 0, 1);
+  0, 1, 0);
```

Comment:
this was a cruel typo, and I am still second-guessing myself as to whether it is a typo. The internal `CC_MARKER` has 17 as its catcode, while a couple of these catcode arrays were expecting it to be 18? Bizarre. I think it may be time to rework the arrays as hashes using the `CC_` names, to avoid typos... Btw this type of fixed-element dispatch is something that I have found Rust's `enum`s and `match` statements to be fantastic for.